org.metaborg.meta.sdf3.ext.statix/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>org.metaborg.meta.sdf3.ext.statix</artifactId>
```

Comment:
Naming consistency concern: the other SDF3 language projects are prefixed with `org.metaborg.meta.lang.template`, referring to the original name of SDF3, "TemplateLang". Maybe we want to use the same scheme for this extension, or rename the other projects to `org.metaborg.meta.lang.sdf3.*`. The latter would be better in my opinion, but more difficult to fix because it will have an impact on basically all Spoofax projects (internal and external).

Reply:
Agreed, that is a good point. Inconsistent naming is probably confusing for users. (Although the fact that we have a dependency on `.template` but we talk about SD3 everywhere is probably also not very clear :P). In the past we have decided against renaming established meta languages because the impact would be too big.